### DISCUSSION TITLE
Are there any plans to support Vulkan and Metal?

### DISCUSSION THREAD

**Discussion by Barugon** [FyroxEngine/Fyrox#310] [Q&A] [answered]
OpenGL is no longer being developed and I'm sure that Apple will eventually drop it from their platforms (if they haven't already), so what's the plan for Vulkan and Metal (if any)? What about using something like [wgpu](https://github.com/gfx-rs/wgpu)?

**Comment by mrDIMAS** [comment, marked as answer]
There are some plans for that, but they're low priority. Current renderer works pretty fine and I have no spare time to do a complete re-write of it, it is very complex task and probably will take few months. If there are volunteers to do that, then go ahead, I will be happy to see new renderer.
